config/locales/nl.yml

```diff
@@ -1,7 +1,10 @@
 nl:
   update: Wijzig
+  sign_in: "Sign in"
+  sign_up: "Sign up"
```

Comment:
Do we need english locales in `nl.yml`? Is there any chance to reuse I18n fallbacks to fallback to english locale for non-translated keys?